env/cloud/pr_review/terragrunt.hcl

```
@@ -0,0 +1,56 @@
+terraform {
+  source = "../../../aws//pr_review"
+}
+
+dependencies {
+  paths = ["../app", "../network"]
+}
+
+dependency "app" {
+  config_path = "../app"
+
+  mock_outputs_allowed_terraform_commands = ["init", "fmt", "validate", "plan", "show"]
+  mock_outputs_merge_with_state           = true
```

The reason the mock outputs aren't being picked up is because a state already exists fro the `app` and `network` modules.  You can force Terragrunt to use the mock values like so:
```suggestion
  mock_outputs_merge_with_state           = true
  mock_outputs_merge_strategy_with_state  = "shallow"
```